src/display/Display.spec.js

```diff
@@ -1 +1,20 @@
-// Test away!
\ No newline at end of file
+// Test away! 
+import React from "react";
+import renderer from "react-test-renderer";
+import { render, fireEvent } from "@testing-library/react";
+import Display from "./Display";  
+
+describe("<Display />", () => {
+    it("Default state set to open and unlocked", () => {
+      const { getByText } = render(<Display />);
+      // we will Check if unlocked and open are displayed on screen
+      expect(getByText("Open"));
+      expect(getByText("Unlocked"));
+    });
+    it("Accepting props", () => {
+      // we will Check if display component receives props and changes based off received props
+      const { getByText } = render(<Display locked={true} closed={true} />);
+      expect(getByText("Closed"));
+      expect(getByText("Locked"));
+    });
+  });
```

Comment:
Here are the requirements for the Display: 

    displays if gate is open/closed and if it is locked/unlocked
    displays 'Closed' if the closed prop is true and 'Open' if otherwise
    displays 'Locked' if the locked prop is true and 'Unlocked' if otherwise
    when locked or closed use the red-led class

The first three are kind of covered here, though they could be more explicitly covered, but the last one isn't.
    when unlocked or open use the green-led class